mailing_list/management/commands/activate_all_lists.py

```diff
@@ -45,10 +46,18 @@ def handle(self, *args, **options):
                 failures[course['id']] = str(e)
                 continue
 
+            # the "is it the primary list" logic varies depending on whether or
+            # not the list is crosslisted.  figure out which logic to use
+            # before bucketing the lists.
+            if any([l['is_course_list'] for l in lists]):
+                is_primary = itemgetter('is_course_list')
+            else:
+                is_primary = itemgetter('is_primary')
```

Comment:
Yeah, that's why we're picking one based on the presence of any course lists.  If there is one, we want it to be the primary.  If not, we want the primary section to be the primary.